public/main.js

```diff
@@ -5,6 +5,39 @@ var todoTitle = document.getElementById("new-todo");
 var error = document.getElementById("error");
 var countLabel = document.getElementById("count-label");
 var clearCompleteButton = document.getElementById("clear-complete-button");
+var filterSelector = document.getElementById("todo-filters");
+
+// Filters for items within the todo list; may not be combined
+var listFilters = {
+    // Shows all items
+    "all": function(item) {
+        return true;
+    },
+    // Shows only incomplete items
+    "active": function(item) {
+        return !item.isComplete;
+    },
+    // Shows only complete items
+    "completed": function(item) {
+        return item.isComplete;
+    }
+};
+
+var currentTodoListFilter = "all";
+
+// Create the list of filter buttons and add them to the page
```

Comment:
Change comment to `Add filters to page as buttons`, as you are no longer creating a list.